app/_data/docs_nav_gateway_3.0.x.yml

```
@@ -74,117 +68,131 @@ items:
-            url: /kong-production/running-kong/start-kong-securely
           - text: Running Kong as a non-root user
-            url: /kong-production/running-kong/kong-user
+            url: /production-deployment/running-kong/kong-user
+          - text: Securing the Admin API
+            url: /production-deployment/running-kong/secure-admin-api
           - text: Configure Kong Gateway and SystemD
-            url: /kong-production/running-kong/systemd
-      - text: Kubernetes
+            url: /production-deployment/running-kong/systemd
+      - text: Access Control
         items:
-          - text: Kubernetes Deployment Options
-            url: /kong-production/kubernetes/kubernetes-deployment-options
+          - text: Start Kong Gateway Securely
+            url: /production-deployment/access-control/start-securely
+          - text: Programatically creating Admins
+            url: /production-deployment/access-control/register-admin-api
+          - text: Enabling RBAC
+            url: /production-deployment/access-control/enable-rbac
       - text: Licenses
         items:
           - text: Overview
-            url: /kong-production/licenses/
-          - text: Access your license
-            url: /kong-production/licenses/access-license
+            url: /licenses/
+          - text: Download your license
+            url: /licenses/download
           - text: Deploy Enterprise License
-            url: /kong-production/licenses/deploy-license
+            url: /licenses/deploy
+          - text: Using the License API
+            url: /licenses/examples
           - text: Monitor Licenses usuage
-            url: /kong-production/licenses/report
+            url: /licenses/report
       - text: Networking
         items:
           - text: Default Ports
-            url: /kong-production/networking/default-ports
+            url: /production-deployment/networking/default-ports
           - text: DNS considerations
-            url: /kong-production/networking/dns-considerations
+            url: /production-deployment/networking/dns-considerations
           - text: Network and Firewall
-            url: /kong-production/networking/network
+            url: /production-deployment/networking/firewall
       - text: Kong Configuration File
-        url: /kong-production/kong-conf
+        url: /production-deployment/kong-conf
       - text: Kong Gateway Environment Variables
-        url: /kong-production/environment-variables
+        url: /production-deployment/environment-variables
       - text: Embedding Kong in Open Resty
-        url: /kong-production/kong-openresty
-      - text: Serving a website and API's from Kong
-        url: /kong-production/website-api-serving
+        url: /production-deployment/kong-openresty
+      - text: Serving a Website and API's from Kong
+        url: /production-deployment/website-api-serving
       - text: Monitoring
         items:
           - text: Introduction
```

```suggestion
          - text: Overview
```
Using this term instead of "Introduction" to match the rest of the nav.